src/main/java/org/gridsuite/modification/server/modifications/LineModification.java

```diff
@@ -99,61 +97,91 @@ private void modifyCharacteristics(Line line, LineModificationInfos lineModifica
             .withDefaultMessage("Characteristics")
             .withSeverity(TypedValue.INFO_SEVERITY)
             .build());
-        if (lineModificationInfos.getSeriesResistance() != null) {
-            characteristicsReporter.report(ModificationUtils.getInstance().buildModificationReportWithIndentation(line.getR(),
-                    lineModificationInfos.getSeriesResistance().getValue(), "Series resistance", 1));
-            line.setR(lineModificationInfos.getSeriesResistance().getValue());
+        Double seriesResistance = lineModificationInfos.getSeriesResistance() != null
+                ? lineModificationInfos.getSeriesResistance().getValue()
+                : null;
+        if (seriesResistance != null) {
```

Comment:
We can check directly here: if (lineModificationInfos.getSeriesResistance() != null)